### DISCUSSION TITLE
How to reflash the bootloader with factory firmware | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#908] [Q&A]
# How to reflash the bootloader with factory firmware | Seeed Studio Wiki

XIAO ESP32C3-FAQ

https://wiki.seeedstudio.com/reflash_the_bootloader/

**Comment by greenpdx** [comment]
I wish you would also put pytool.py commandline for Linux to flash a new bootloader

**Comment by davidgs** [comment]
`esptool.py --chip esp32c3 --before no_reset --port /dev/cu.usbmodem83201  write_flash -z --flash_mode dio --flash_freq 80m --flash_size 8MB 0x0 ~/Downloads/ESP32-C3_RFTest_108_2b9b157_20211014.bin`

**Comment by alanjose10** [comment]
Can this be done for nRF52840? I have a bricked seeed xiao nRF52840 and would like to try and reflash the bootloader.

**Reply by nfs0619** [reply]
To reflash the bootloader for the Seeed XIAO nRF52840, a J-Link is required. The flashing process is as follows:
https://wiki.seeedstudio.com/XIAO_BLE/#access-the-swd-pins-for-debugging-and-reflashing-bootloader
